golem-api-grpc/proto/golem/rib/expr.proto

```diff
@@ -175,7 +175,7 @@ message CallExpr {
 
 message InvocationName {
    oneof name {
-     golem.rib.ParsedFunctionName parsed = 1;
+     golem.rib.DynamicParsedFunctionName parsed = 1;
```

Comment:
how? These are grpc types only for internal persistence.

Reply:
Ok, discard my question. In terms of backward compatibility, well the function name is different now because has `Exprs` in it, and it's a different structure. I need to test backward compatibility.

Reply:
What is internal persistence? If it is stored in any database, then this is a breaking change.

Reply:
Need to test/fix backward compatibility , which is what is remaining.

Reply:
Another way to handle this would be to detect breakage and recompile from source. Since the IR is not exposed to the user.